src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -5040,6 +5061,16 @@ public override void VisitForEachIterationVariables(BoundForEachStatement node)
             // declare and assign all iteration variables
             TypeWithAnnotations sourceType = node.EnumeratorInfoOpt?.ElementTypeWithAnnotations ?? default;
             TypeWithState sourceState = sourceType.ToTypeWithState();
+
+#pragma warning disable IDE0055 // Fix formatting
```

Comment:
is there something wrong with the formatting in here? It seems fine..

Perhaps it wants the `{` on the same indent level as the `var`?

Reply:
It is a known issue with the formatter.

---
In reply to: [274530915](https://github.com/dotnet/roslyn/pull/34926#discussion_r274530915) [](ancestors = 274530915)